### DISCUSSION TITLE
Would love to collaborate on working on a standard for multi token vaults

### DISCUSSION THREAD

**Discussion by 0xvanbeethoven** [superform-xyz/experimental-4626#17] [General]
Hey guys, I'm a smart contract dev at Sorella Labs. We're developing smart contract vaults with similar functionality however we calculate exchange rate differently and allow for multiple token deposits and withdraws in any proportion. I think your vault is great and would love to work together and collaborate on a standard. 

Lmk my email is: vanbeethoven@sorellacapital.com

**Comment by shortdoom** [comment]
Hey,

What do you guys have in mind? We didn't update this repository for some time but its a recurring topic to get back into multiVault ERC4626 development. Feel free to join superform discord and ping one of the protocol admins there if you want to set some meet / talk it further.

https://discord.gg/superform

**Comment by Armankhan566** [comment]
Why matterlabs block me on github please unblock me
